lib/liquid/context.rb

```diff
@@ -47,6 +48,14 @@ def add_filters(filters)
       @strainer = nil
     end
 
+    def add_global_filters(global_filters)
```

Comment:
Also, `add` implies that you are adding to a list.  An attribute writer like `attr_writer :global_filter` or `def global_filter=(global_filter)` would be more appropriate.